Airport Ticket Booking/Airport Ticket Booking/Commands/PassengerCommands/CancelBookingCommand.cs

```diff
@@ -0,0 +1,23 @@
+﻿using AirportTicketBooking.DBHandler;
+
+namespace AirportTicketBooking.Commands.PassengerCommands
+{
+    internal class CancelBookingCommand : ICommandPassenger
+    {
+        public List<object> Execute(string userEmail, string[] parameters, FlightDataHandler flightDataHandler, BookingDataHandler bookingDataHandler)
+        {
+            (string parameterName, string flightID) = FetchParameters(parameters);
+            if (!parameterName.Equals("flightid"))
```

Comment:
Input Validation inside the Parsing Function, and The Domain/Model validation inside its own. function. that would make it more readable.